### PR TITLE
Documentation 2021c

### LINKED ISSUE #9111: Documentation 2021c
Issue overview
------

A place to collect requested documentation fixes. Add a new comment to the issue with new doc-change-only issues. Someone will eventually fix them all up at the same time once critical mass has been reached.

### PR DESCRIPTION
**Pull request by jcyuan2020**
Pull request overview
---------------------
 - Fixes #9111

**NOTE: ENHANCEMENTS MUST FOLLOW A SUBMISSION PROCESS INCLUDING A FEATURE PROPOSAL AND DESIGN DOCUMENT PRIOR TO SUBMITTING CODE**

### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [x] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish
 - [ ] Pull requests that impact EnergyPlus code must also include unit tests to cover enhancement or defect repair
 - [ ] Author should provide a "walkthrough" of relevant code changes using a GitHub code review comment process
 - [ ] If any diffs are expected, author must demonstrate they are justified using plots and descriptions
 - [ ] If changes fix a defect, the fix should be demonstrated in plots and descriptions
 - [ ] If any defect files are updated to a more recent version, upload new versions here or on DevSupport
 - [ ] If IDD requires transition, transition source, rules, ExpandObjects, and IDFs must be updated, and add IDDChange label
 - [ ] If structural output changes, add to output rules file and add OutputChange label
 - [ ] If adding/removing any LaTeX docs or figures, update that document's CMakeLists file dependencies

### Reviewer
This will not be exhaustively relevant to every PR.
 - [ ] Perform a Code Review on GitHub
 - [ ] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [ ] CI status: all green or justified
 - [ ] Check that performance is not impacted (CI Linux results include performance check)
 - [ ] Run Unit Test(s) locally
 - [ ] Check any new function arguments for performance impacts
 - [ ] Verify IDF naming conventions and styles, memos and notes and defaults
 - [ ] If new idf included, locally check the err file and other outputs

### CODE REVIEW

**Comment by nrel-bot-3** [comment]
@jcyuan2020 @myoldmopar it has been 28 days since this pull request was last updated.

**Comment by Myoldmopar** [comment]
@jcyuan2020 are you done making commits here?  If so I'll build locally and poke through the changes.

File: doc/engineering-reference/src/advanced-surface-concepts/exterior-naturally-vented-cavity.tex [hunk 1]
```diff
@@ -205,7 +205,7 @@ \subsection{Convection Coefficients}\label{convection-coefficients}
 Exterior cavity modeling requires calculating up to three different coefficients for surface convection heat transfer.~ These coefficients are defined in the classic way by:
 
 \begin{equation}
-{h_c} = \frac{{{T_{air}} - {T_{surf}}}}{{{{q''}_{conv}}}}
```
Comment (Myoldmopar) [inline doc/engineering-reference/src/advanced-surface-concepts/exterior-naturally-vented-cavity.tex:208]: :+1:

File: doc/engineering-reference/src/simulation-models-encyclopedic-reference-003/zone-internal-gains.tex [hunk 2]
```diff
@@ -23,7 +23,7 @@ \subsection{Heat Gain from Lights}\label{heat-gain-from-lights}
 
 \(MassFlowRA\) is the mass flow rate at the return node (kg/s)
 
-\(CpAir\) is the specific heat (J/kg)
+\(CpAir\) is the specific heat (J/kg-K)
```
Comment (Myoldmopar) [inline doc/engineering-reference/src/simulation-models-encyclopedic-reference-003/zone-internal-gains.tex:26]: Yep, makes sense.

File: doc/engineering-reference/src/simulation-models-encyclopedic-reference-005/occupant-thermal-comfort.tex [hunk 3]
```diff
@@ -227,15 +227,15 @@ \subsubsection{Description of the model and algorithm}\label{description-of-the-
 \end{equation}
 
 \begin{equation}
-{Q_{r}}=3.96*f\left(_{l}\right)*{[}(0.01*T\left(_{a}\right))\left(^{4}\right)-(0.01*T\left(_{a}\right))\left(^{4}\right){]}
+{Q_{r}}=3.96*f_{cl}*{[}(0.01*T_{cla})^{4}-(0.01*T_{ra})^{4}{]}
```
Comment (Myoldmopar) [inline doc/engineering-reference/src/simulation-models-encyclopedic-reference-005/occupant-thermal-comfort.tex:230]: Interesting, OK.  Definitely cleaner, but I think it's changing the equation, right?  I'll have to peek at before/after.

File: doc/input-output-reference/src/input-for-output.tex [hunk 4]
```diff
@@ -653,15 +653,16 @@ \subsection{Output:EnvironmentalImpactFactors}\label{outputenvironmentalimpactfa
...
 \toprule
+%Electricity:Facility & Diesel:Facility \tabularnewline
+%\midrule
+%\endfirsthead
+%
+%\toprule
+%Electricity:Facility & Diesel:Facility \tabularnewline
+%\midrule
+%\endhead
+%
```
Comment (Myoldmopar) [inline doc/input-output-reference/src/input-for-output.tex:664]: Why?

File: doc/input-output-reference/src/input-for-output.tex [hunk 5]
```diff
@@ -2068,6 +2069,24 @@ \subsubsection{Outputs}\label{outputs-029}
 
 What shows up in the output are the Energy Meters that were utilized in this particular input file, the pollutants, and the Total Carbon Equivalent.
 
+\subsection{Output:JSON}\label{outputjson}
```
Comment (Myoldmopar) [inline doc/input-output-reference/src/input-for-output.tex:2072]: :+1:

File: idd/Energy+.idd.in [hunk 6]
```diff
@@ -86799,8 +86798,7 @@ ZoneHVAC:RefrigerationChillerSet,
 
 DemandManagerAssignmentList,
        \extensible:2 Duplicate the last two fields DemandManager Object Type and DemandManager Name
-       \memo a list of meters that can be reported are available after a run on
-       \memo the meter dictionary file (.mdd) if the Output:VariableDictionary has been requested.
+       \memo Demand Manager Assignment List
```
Comment (Myoldmopar) [inline idd/Energy+.idd.in:86801]: How is this a more helpful memo?  Same with the next one.

File: doc/input-output-reference/src/overview/group-radiative-convective-units.tex [hunk 7]
```diff
@@ -54,7 +54,7 @@ \subsubsection{Inputs}\label{inputs-038}
 
 \paragraph{Field: Rated Average Water Temperature}\label{field-rated-averagewatertemperature}
 
-This field is the rated average water temperature for the baseboard heater which is published in the manufacturer's literature in degree Celsius. It typically ranges from 65.56°C to 115.36°C in the I = B = R rating document while the lowest allowable temperature is 32.22°C. The default value is 87.78°C. If the user does not enter this field, the default value is assumed.
+This field is the rated average water temperature for the baseboard heater which is published in the manufacturer's literature in degree Celsius. It typically ranges from 65.56$^\circ$C to 115.36$^\circ$C in the I = B = R rating document while the lowest allowable temperature is 32.22$^\circ$C. The default value is 87.78$^\circ$C. If the user does not enter this field, the default value is assumed.
```
Comment (Myoldmopar) [inline doc/input-output-reference/src/overview/group-radiative-convective-units.tex:57]: Good change.  We should define a degree macro if we don't already have one, but that can be a different time.

File: doc/output-details-and-examples/src/output-files/eplustbl-lt-ext-gt.tex [hunk 8]
```diff
@@ -1279,7 +1279,7 @@ \subsection{HVAC Sizing Summary}\label{hvac-sizing-summary}
...
+\emph{System Sizing Method Concurrence} This field applies to ``AirLoop'' coils.  It describes if the central air system sizing was done using ``Coincident'' or ``Non-Coincident'' method. This refers to how the ideal loads sizing routines process the time sequence of zone loads.  The concurrent sum of peaks, or coincident or "block", is often lower than the sum of the individual peaks regardless of when they occur, or non-coincident sum.  This field is not applicable to ``Zone Equipment'' coils and will show ``N/A'' here.  This corresponds to the input field called Type of Zone Sum to Use in the Sizing:System input object. Text string.
```
Comment (Myoldmopar) [inline doc/output-details-and-examples/src/output-files/eplustbl-lt-ext-gt.tex]: This line has a quotation mark character that should be converted.  We should most likely do an intelligent search/replace in the code to clean these up.  If you have changes to make for this PR, you might go ahead and clean this up, but it's not a big deal.

**Review by Myoldmopar — COMMENTED** [review]
I think there might still be a commit to make here.  I'm a bit confused about the additions to the table contents yet they are commented.  And I'm not sure the IDD changes are actually helpful.  Please respond to my comments and we'll get this pushed along.  Thanks @jcyuan2020 !

**Comment by jcyuan2020** [comment]
> 

I think it addressed the documentation issues brought up in the original post and plus a few more recent ones. There one item about whether the end use category `DHW` is a valid entry, but it seems the comments followed mean that it should be left as it is. 

Thanks for reviewing this! I will respond to the detailed comments shortly after.

File: idd/Energy+.idd.in (hunk 6 above)
Comment (jcyuan2020) [reply idd/Energy+.idd.in:86801]: Probably need a better sentence to explain what this object is--I will dig out a little bit more from the IORef descriptions to come up with a proper sentence.

File: doc/output-details-and-examples/src/output-files/eplustbl-lt-ext-gt.tex (hunk 8 above)
Comment (jcyuan2020) [reply doc/output-details-and-examples/src/output-files/eplustbl-lt-ext-gt.tex]: Ok. I will change this one.

File: doc/input-output-reference/src/input-for-output.tex (hunk 4 above)
Comment (jcyuan2020) [reply doc/input-output-reference/src/input-for-output.tex:664]: The basic idea here is that for this table only a top rule is needed and then the two columns. 
However, it is indeed a little perplex in the original form why it need two sets: one if from Line 655 to Line 658, ending with `endfirsthead`; and the second set from Line 660 to Line 663, ending with `endhead`---but everything else is the same. Maybe some specification in the `longtable`, but here I jsut need the outer frame (so only top rule) would be good--so a simplification is made here to keep only two lines.

File: idd/Energy+.idd.in (hunk 6 above)
Comment (Myoldmopar) [reply idd/Energy+.idd.in:86801]: Yes, better, thanks.

**Comment by Myoldmopar** [comment]
Thanks @jcyuan2020.  I made one final tweak to capitalize a letter in the IDD, but otherwise, this is a fine set of improvements.  Pushing my changes and merging without waiting on CI.

**Pull request merged**